lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -938,26 +555,16 @@ UsdAttributeEnumString::create(const UsdSceneItem::Ptr& item, const PXR_NS::UsdA
 std::string UsdAttributeEnumString::get() const
 {
     PXR_NS::VtValue vt;
-    if (UsdAttribute::get(vt, getCurrentTime(sceneItem()))) {
-        if (vt.IsHolding<TfToken>()) {
-            TfToken tok = vt.UncheckedGet<TfToken>();
-            return tok.GetString();
-        } else if (vt.IsHolding<std::string>()) {
-            return vt.UncheckedGet<std::string>();
-        }
+    if (UsdAttribute::get(vt, getCurrentTime(sceneItem())) && vt.IsHolding<std::string>()) {
+        return vt.UncheckedGet<std::string>();
```

Comment:
This class wraps a std::string USD datatype. No need to check what the Vt is holding.